cpp/include/cudf/strings/detail/split_utils.cuh

```diff
@@ -22,6 +22,31 @@ namespace cudf {
 namespace strings {
 namespace detail {
 
+constexpr bool is_whitespace(char_utf8 ch) { return ch <= ' '; }
```

Comment:
Why does this checking "is whitespace" return true for all characters `<= ' '`? Why not just `ch == ' '`?

Reply:
Space is not the same as whitespace. The `<= ' '` accounts for new-lines, tabs, carriage returns, etc.